packages/common/src/groups/_internal/GroupBusinessRules.ts

```diff
@@ -74,4 +75,10 @@ export const GroupPermissionPolicies: IPermissionPolicy[] = [
     permission: "hub:group:workspace:members",
     dependencies: ["hub:group:view"],
   },
+  {
+    permission: "hub:group:shareContent",
+    dependencies: ["hub:group"],
+    authenticated: true,
+    privileges: ["portal:user:shareToGroup"],
```

Comment:
I think you also need to verify the user is a member of the group. We can do this with `assertions`...
```
assertions:[
  {  
      property: "context:currentUser",
      type: "is-group-member",
      value: "entity:id",
  }
]
```